src/py42/_internal/clients/alertrules/file_type_mismatch.py

```diff
@@ -0,0 +1,26 @@
+import json
+
+from py42.clients import BaseClient
+
+
+class FileTypeMisMatch(BaseClient):
```

Comment:
Should we name these ending with the word `client`, as in our convention?
It does not matter too much since they are internal

Reply:
Renamed.